### PR TITLE
[11.0] [MIG] purchase_location_by_line: Migration to 11.0

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Superseed: #512 
Depends on: #584

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 server-tools
+purchase-workflow https://github.com/Eficent/purchase-workflow.git 11.0-mig-purchase_delivery_split_date
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: dependency has ben merged :tada: Can you rebase and delete this?

Comment (HviorForgeFlow) [reply oca_dependencies.txt]: Done

**Review by LoisRForgeFlow — APPROVED** [review]
Functional test :+1:

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Works as expected.

**Pull request merged**